src/main/java/net/fabricmc/loader/impl/launch/FabricMixinBootstrap.java

```diff
@@ -79,7 +73,53 @@ public static void init(EnvType side, FabricLoaderImpl loader) {
 		}
 
 		MixinBootstrap.init();
-		getMixinConfigs(loader, side).forEach(FabricMixinBootstrap::addConfiguration);
+
+		try {
+			Method addConfigurationFabric = Mixins.class.getMethod("addConfigurationFabric", String.class, Map.class);
```

Comment:
Could do this calling the method directly then catching `NoSuchMethodError`?

Reply:
I don't want to risk it, AFAIK the JVM spec doesn't guarantee that it'd be safe and this is good enough.